scripts/map-randomizer.js

```diff
@@ -92,5 +98,9 @@
   });
 
   document.getElementById('again').addEventListener('click', render);
+  document.forms[0].addEventListener('submit', function (event) {
```

Comment:
I would just assign `render` here, and have the `render` function deal with an option event argument. That keeps the code cleaner and eliminates an unnecessary function execution.